Resolve linter issues (ansible-lint, yamllint, flake8)

ThomasSanson opened the pull request:
Close #308

vitabaks commented:
@ThomasSanson consider offering a PR for yedit.py on the project [repository](https://github.com/kwoodson/ansible-role-yedit)

ThomasSanson commented:
Hi @vitabaks, thank you for your comment. 

I'm not quite sure if I understood your request correctly. 

Are you suggesting that I should I cancel the changes I made in the current Pull Request ? 

Please let me know so I can proceed accordingly. Thanks !

vitabaks commented:
not necessarily cancel these changes.

It would just be nice to suggest improvements to the original repository of this module as well.

vitabaks commented:
in fact, this is not a role, but a library for managing the yaml configuration, which you can observe here https://github.com/vitabaks/postgresql_cluster/tree/master/roles/patroni/library

ThomasSanson commented:
@vitabaks, I'm back from vacation ^^

I have just made the corrections for the merge conflicts, and everything is good to go for me to merge this pull request.

vitabaks commented:
@ThomasSanson Welcome back! I'm glad you're here)

please don't forget to remove the exceptions from the [.ansible-lint](https://github.com/vitabaks/postgresql_cluster/blob/master/.ansible-lint) based on what has been fixed. 

To have a clear understanding of what else needs attention.

ThomasSanson commented:
Hi @vitabaks,

Thank you for your comment.

I understand your concern regarding the skip_list rules in .ansible-lint. For now, I have just made the linters pass for this pull request.

My plan is to create a separate issue for each rule in skip_list after this pull request. This will allow us to address each issue systematically and have a clear understanding of what else needs attention.

I hope this addresses your concerns. Thank you again for your feedback.

vitabaks commented on config_pgcluster.yml:
```diff
@@ -133,18 +133,20 @@
   tags: config_balancers,load_balancing, haproxy
 
-- hosts: postgres_cluster
+- name: config_pgcluster.yml | Deploy Postgres Cluster
```
```suggestion
- name: config_pgcluster.yml | Configure Postgres Cluster
```

vitabaks commented on deploy_pgcluster.yml:
```diff
@@ -1,13 +1,12 @@
 ---
-
-- name: Deploy PostgreSQL HA Cluster (based on "Patroni" and "{{ dcs_type }}")
+- name: deploy_pgcluster.yml | Deploy PostgreSQL HA Cluster (based on "Patroni" and "{{ dcs_type }}")
```
```suggestion
- name: Deploy PostgreSQL HA Cluster (based on "Patroni" and "{{ dcs_type }}")
```
why is the playbook name added here?

ThomasSanson replied:
I have incorporated the playbook names into the task names to enhance readability in the logs. However, it's important to note that this change is a proactive step towards our planned migration to collection mode
